fuel-merkle/src/common/position.rs

```diff
@@ -374,20 +388,20 @@ mod test {
 
     #[test]
     fn test_left_child() {
-        assert_eq!(Position(7).left_child(), Position(3));
-        assert_eq!(Position(3).left_child(), Position(1));
-        assert_eq!(Position(1).left_child(), Position(0));
-        assert_eq!(Position(11).left_child(), Position(9));
-        assert_eq!(Position(9).left_child(), Position(8));
+        assert_eq!(Position(7).child(Side::Left), Position(3));
+        assert_eq!(Position(3).child(Side::Left), Position(1));
+        assert_eq!(Position(1).child(Side::Left), Position(0));
+        assert_eq!(Position(11).child(Side::Left), Position(9));
+        assert_eq!(Position(9).child(Side::Left), Position(8));
```

Comment:
Is this necessary? It looks like we still have the higher-level function `left_child` (and similarly below, `right_child`)

Reply:
Not anymore, as I re-added those. Restored in fcf38742.